task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,7 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+  return !(num % 3) && !(num % 5) ? 'FizzBuzz' : !(num % 5) ? 'Buzz' : !(num % 3) ? 'Fizz' : num;
```

Comment:
It's more readable if you use if-statements instead of multiple ternary operators in a row.